src/dispatcher.js

```diff
@@ -95,6 +95,10 @@ export default class Dispatcher {
     .on('paste.editable', selector, function (event) {
       const element = this
 
+      event.preventDefault()
+      const textData = event.originalEvent.clipboardData.getData('text/plain')
+      self.$document.get()[0].execCommand('insertHTML', false, textData)
```

Comment:
doesn't `self.$document.get(0)`  do the same as `.get()[0]`?

Reply:
Ah, yes, right, thanks!